script.js

```diff
@@ -0,0 +1,67 @@
+let book = [];
+
+const givenName = document.querySelector('#name');
+const givenAuthor = document.querySelector('#author');
+const btnClass = document.querySelector('#Button');
+const listOfName = document.querySelector('#listOfName');
+
+function updateStorage() {
+  localStorage.setItem('books', JSON.stringify(book));
+}
+function removeBook(id, c = true) {
+  if (c === false) {
+    return;
+  }
+  for (let i = 0; i < book.length; i += 1) {
+    const bookid = i;
+    if (book[bookid].id === Number(id)) {
+      book.splice(bookid, 1);
```

Comment:
- [ ] [OPTIONAL] Nice Job removing a book using the splice method but I kindly suggest that you use the Array.filter method because it is best practice to use the filter method as the slice mutates the array but the filter method doesn't

Reply:
@topeogunleye thank you very much coach im going to change it

Reply:
You're welcome 👌👌